device_services/generic/chargepoint.md

```diff
@@ -8,38 +8,55 @@ Chargepoint service is used to represent EV chargers.
+| in   | cmd.charge.stop                 | null       |                                                                     | Stops/pauses charging an EV. Charger must be in `charging` state.                                                      |
+| -    |                                 |            |                                                                     |                                                                                                                        |
+| in   | cmd.state.get_report            | null       |                                                                     | Gets the `state` of the chargepoint.                                                                                   |
+| out  | evt.state.report                | string     | `charging_mode`                                                     | Reports the `state` of the chargepoint, see [`sup_states`](#service-properties) for list of possible states.           |
+| -    |                                 |            |                                                                     |                                                                                                                        |
+| in   | cmd.cable_lock.set              | bool       |                                                                     | Locks and unlocks the cable/connector.                                                                                 |
+| in   | cmd.cable_lock.get_report       | null       |                                                                     | Gets the status of the cable/connector lock.                                                                           |
+| out  | evt.cable_lock.report           | bool       |                                                                     | Reports `true` if the cable/connector is **locked** and `false` otherwise.                                             |
+| -    |                                 |            |                                                                     |                                                                                                                        |
+| in   | cmd.current_session.set_current | int        |                                                                     | Sets the offered current for the ongoing session in `A`, must be an integer between `6` and a set `max_current` value. |
+| in   | cmd.current_session.get_report  | null       |                                                                     | Requests energy consumed during the current session and optionally additional characteristics.                         |
+| out  | evt.current_session.report      | float      | `previous_session`, `started_at`,  `finished_at`, `offered_current` | Reports energy consumed during the current session in `kWh` and optionally additional characteristics.                 |
```

Comment:
if we decided to split, maybe we should put the `previous_session` and `finished_at` properties under `evt_previous_session.report` with `finished_at` property and not overload **current**`_session` info with **previous**`_session` info?

Reply:
`finished_at` is an attribute of the CURRENT SESSION. 

You start charging, there is only started_at.
You stop charging, there is started_at and stoped at. Session is finished but is still the last session, so it's also a current session. 

Use case: A customer plugs his car at 11 PM an goes to bed. Car starts charging at 11 PM and charges until 2 AM when the session finishes because car is fully charged. In the morning the customer looks on mobile app and sees:

Status: Finished
Started at: 11 PM
Finished at: 2 AM